kbchat/kbchat_test.go

```diff
@@ -140,45 +166,209 @@ func TestMain(m *testing.M) {
 }
 
 func TestGetUsername(t *testing.T) {
-	require.Equal(t, kbc.GetUsername(), config.Bots.Alice1.Username)
+	require.Equal(t, alice.GetUsername(), config.Bots.Alice1.Username)
```

Comment:
i would recommend re-writing these tests so that each one can be run in isolation. i should be able to run just `TestGetUsername` to know if `GetUsername` is busted. and it should do whatever setup it needs to do to ensure that it's working. we can extract that into non-test setup methods.